hail/python/hail/methods/statgen.py

```diff
@@ -1983,6 +1998,446 @@ def _linear_skat(group,
     return ht.select_globals('y_residual', 's2', 'n_complete_samples')
 
 
+@typecheck(group=expr_any,
+           weight=expr_float64,
+           y=expr_float64,
+           x=expr_float64,
+           covariates=sequenceof(expr_float64),
+           max_size=int,
+           null_max_iterations=int,
+           null_tolerance=float,
+           accuracy=numeric,
+           iterations=int)
+def _logistic_skat(group,
+                   weight,
+                   y,
+                   x,
+                   covariates,
+                   max_size: int = 46340,
+                   null_max_iterations: int = 25,
+                   null_tolerance: float = 1e-6,
+                   accuracy: float = 1e-6,
+                   iterations: int = 10000):
+    r'''The logistic sequence kernel association test (SKAT).
+
+    Logistic SKAT tests if the phenotype, `y`, is significantly associated with the genotype,
+    `x`. For :math:`N` samples, in a group of :math:`M` variants, with :math:`K` covariates, the
+    model is given by:
+
+    .. math::
+
+        \begin{align*}
+        X &: R^{N \times K} \\
+        G &: \{0, 1, 2\}^{N \times M} \\
+        \\
+        \textrm{logit}(P(y=1)) &= \beta_0 X + \beta_1 G + \varepsilon \quad\quad \varepsilon \sim N(0, \sigma^2)
```

Comment:
This is still saying the residuals are normally distributed. I think the right way to express the logistic regression model is
```
y ~ Bernoulli(logit^{-1}(beta_0 X + beta_1 G))
```